examples/democoin/app/app.go

```diff
@@ -65,12 +65,15 @@ func NewDemocoinApp(logger log.Logger, db dbm.DB) *DemocoinApp {
 		&types.AppAccount{}, // prototype
 	).Seal()
 
+	// Generate codespacer
+	codespacer := sdk.NewCodespacer()
```

Comment:
I think we can generate this automatically with the Baseapp.

Reply:
Added in `NewBaseApp`